### PR TITLE
done with slider component

### PR DESCRIPTION
**Pull request by Varun-28**
Done with simple slider component.

- consists of all required attributes

slider

### CODE REVIEW

**Review by Syedmha — COMMENTED** [review]
Looks good to merge.

**Review by OmKakatkar — APPROVED** [review]
Like the PR is small and exactly for the intended purpose only.
Looks good to merge

File: css/component_styles/slider.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 @import url('./components.css');
 
-/* -------- slider css -------- */
\ No newline at end of file
+/* -------- slider css -------- */
```
Comment (utkarshducoder) [inline css/component_styles/slider.css:3]: try to change the colour of slider and remove the comment part

Comment (Varun-28) [reply css/component_styles/slider.css:3]: But I want it to be of that colour as the user can change its colour as per need and I don't see any issue with the comment to be removed

**Review by parulgupta2001 — COMMENTED** [review]
Only one suggestion, use class instead of id wherever possible. Otherwise everything looks ok.

**Pull request merged**